packages/date-picker/src/vaadin-infinite-scroller.js

```diff
@@ -186,9 +186,8 @@ class InfiniteScroller extends PolymerElement {
     }
 
     // Check if we scrolled enough to translate the buffer positions.
-    const bufferOffset = this.root.querySelector('.buffer').offsetTop;
-    const upperThresholdReached = scrollTop > this._buffers[1].translateY + this.itemHeight + bufferOffset;
-    const lowerThresholdReached = scrollTop < this._buffers[0].translateY + this.itemHeight + bufferOffset;
+    const upperThresholdReached = scrollTop > this._buffers[1].translateY + this.itemHeight + this.bufferOffset;
```

Comment:
nit: we could use something like this to avoid checking `offsetTop` twice:

```js
const offset = this.itemHeight + this.bufferOffset;
const upperThresholdReached = scrollTop > this._buffers[1].translateY + offset;
const lowerThresholdReached = scrollTop < this._buffers[0].translateY + offset;
```

Reply:
Done.